client/js/components/Query/StatsBox/DistChart.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import Numeral from "numeral";
+import {BarChart, XAxis, YAxis, Bar, CartesianGrid, Tooltip, ResponsiveContainer} from "recharts";
+
+export default class DistChart extends React.Component {
+
+  formatTick(value) {
+    return Numeral(value).format("0,0")
+  }
+
+  render() {
+    var data = this.props.data.map(function(object, i){
+      var midpoint = (object.upper + object.lower)/2;
+      midpoint = Number(midpoint.toFixed(2));
+      return { x: midpoint, y: object.count}
+    })
+    return <ResponsiveContainer height={300}>
+             <BarChart data={data}>
+               <XAxis dataKey="x"
+                      tickCount={10}
+                      type="number"
+                      domain={[0, 'dataMax']}
```

Comment:
I don't think we can just use `dataMin`. As I recall it ends up placing a bar right on the y-axis. However, we could find the min and then add a bit of padding to it.